allensdk/brain_observatory/behavior/behavior_ophys_session.py

```diff
@@ -305,6 +306,21 @@ def segmentation_mask_image(self) -> Image:
     def segmentation_mask_image(self, value):
         self._segmentation_mask_image = value
 
+    @property
+    def eye_tracking_data(self) -> pd.DataFrame:
+        """A dataframe containing DeepLabCut eye tracking ellipse fit
+        parameters for eye, pupil, and corneal reflection.
```

Comment:
I'd suggest rewording this part of the docstring slightly to be a little more precise about what this represents (it's not actually the output from DeepLabCut - it contains ellipse fit parameters *derived from* the DeepLabCut model output). How about "A dataframe containing ellipse fit parameters for the eye, pupil and corneal reflection. Fits are derived from tracking points from a DeepLabCut model applied to video frames of the mouse's right eye. Raw tracking points and raw video frames are not exposed by the SDK"